Story-Chief/technical-interview-08-05-2024
# Andrew cauchi

## Description

1. Fix related to the grid structure when the last item card on the Sloths Board is pressed, where x5 columns in the grid are applied instead of x3 columns.
2. Fix to make sure that an item card is properly unchecked when previously checked on the Sloths Board, the counter of selected items will now also update properly.

## Closes issue(s)



## Testing Instructions

To test no. 1 above
1. Open any item card by pressing 'View Details' - this will open side menu with content
2. The grid will re-position itself to fit better on the screen with x3 columns
3. Click on 'View Details' for the last card element on the list - this will open the related content
4. The grid should still fit well with x3 columns

To test no. 2 above
1. Select 1 or more elements from the list of cards on the Sloths board - you will see the number of items selected
2. Deselect 1 or more elements from the list by clicking again on the checkbox
3. The deselected cards should be now unchecked and the list of items selected should have the real number of selected items

## Screenshots
